react-frontend/src/components/Learning/podcast.js

```diff
@@ -0,0 +1,107 @@
+
+import { ContentBlockContainer } from '../StyleComponents/pageContent';
+import { SubHeading, SubSubHeading } from '../StyleComponents/headings';
+import { HrefLinkStandalone } from '../StyleComponents/htmlTags';
+
+const podcastImage = require('../../images/pngIllustrations/podcastGray.png')
+  .default;
+
+function Podcast() {
+  const strapiMediaUrl = useContext(AppConfigContext)['state'][
+    'strapiMediaUrl'
+  ];
+  return (
+    <div>
+      <PodcastBanner />
+      <ContentBlockContainer id="courses">
+        <Row>
+          <Col sm={12}>
+            <SubHeading style={{ display: 'inline-block' }}>
+              Recent Episodes
+            </SubHeading>
+            <HrefLinkStandalone
+              href="https://bcdevexchange.libsyn.com/"
+              target="_blank"
+              rel="noopener noreferrer"
+              style={{ float: 'right' }}
+            >
+              View all {' >'}
+              <FontAwesomeIcon
+                icon={faExternalLinkAlt}
+                style={{ paddingLeft: '5px' }}
+              />
+            </HrefLinkStandalone>
+          </Col>
+        </Row>
+        {/* PODCASTS_QUERY returns the 3 most recent episodes in Strapi  */}
+        <Query query={PODCASTS_QUERY}>
+          {({ data: { podcasts } }) => {
+            console.log(podcasts);
+            return (
+              <div>
+                {podcasts.map((podcast, i) => {
+                  return (
+                    <Row style={{ marginBottom: '16px' }} key={i}>
+                      <Col xs={12} md={4} style={{ height: '200px' }}>
+                        <img
+                          src={strapiMediaUrl + podcast?.CoverImage?.url}
+                          style={{ maxWidth: '100%', height: '100%' }}
+                        />
+                      </Col>
+                      <Col xs={12} md={8}>
+                        <div
+                          style={{
+                            display: 'flex',
+                            flexDirection: 'column',
+                            height: '100%',
+                          }}
+                        >
+                          <p style={{ marginBottom: '0' }}>
+                            {podcast.EpisodeNumber
+                              ? `Episode ${podcast.EpisodeNumber} `
+                              : ``}
+                            &#183; {dateFormat(podcast.AirDate, 'mmmm d,yyyy')}
+                          </p>
+                          <SubSubHeading>{podcast.Title}</SubSubHeading>
+                          <p>{podcast.Description}</p>
+                          <HrefLinkStandalone
+                            href={podcast.Link}
+                            target="_blank"
+                            rel="noopener noreferrer"
+                            style={{ marginTop: 'auto' }}
+                          >
+                            Listen on Libsyn
+                            <FontAwesomeIcon
```

Comment:
Though **"View all"** _is_ an external link, in this case, because we're using the chevron (>), we don't want the external link icon too:

![external-link](https://user-images.githubusercontent.com/16946297/124505060-8c558f00-dd7d-11eb-96bf-b661f3a9a71b.png)

Reply:
I've removed the chevron and run it by Inderdeep, it keeps it in line with the BC Gov design system and doesn't look too crowded.